pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -472,42 +480,58 @@ func getPidsForProcess(name, pidFile string) ([]int, error) {
 }
 
 // Ensures that the Docker daemon is in the desired container.
-func ensureDockerInContainer(cadvisor cadvisor.Interface, oomScoreAdj int, manager *fs.Manager) error {
-	pids, err := getPidsForProcess(dockerProcessName, dockerPidFile)
-	if err != nil {
-		return err
+func ensureDockerInContainer(dockerVersion semver.Version, oomScoreAdj int, manager *fs.Manager) error {
+	type process struct{ name, file string }
+	dockerProcs := []process{{dockerProcessName, dockerPidFile}}
+	if dockerVersion.GTE(containerdVersion) {
+		dockerProcs = append(dockerProcs, process{containerdProcessName, ""})
```

Comment:
I decided to try one more search, and found it! `/run/docker/libcontainerd/docker-containerd.pid` I can't find any documentation of that though...